docs/SEMANTIC_CONVENTIONS.md

```diff
@@ -0,0 +1,127 @@
+| recurrence.interval                            | Describing the interval with which the event recurs                                                                             | text                     | OPTIONAL                    |
+| accessibility.accessibleEntry                  | If the entry/exit is accessible                                                                                                 | boolean                  | OPTIONAL                    |
+| accessibility.accessibleSeats                  | If there are wheelchair places available on the event hall                                                                      | boolean                  | OPTIONAL                    |
+| accessibility.accessibleToilets                | If there are accessible toilets available                                                                                       | boolean                  | OPTIONAL                    |
+| accessibility.accessibleAktivpassLinz          | If the Aktivpass Linz may be applicable                                                                                         | boolean                  | OPTIONAL                    |
+| accessibility.accessibleKulturpass             | If the Kulturpass may be applicable                                                                                             | boolean                  | OPTIONAL                    |
+| concert.genre                                  | Genre of the concert                                                                                                            | text                     | OPTIONAL                    |
+| concert.bandlist                               | List of all bands playing                                                                                                       | list                     | OPTIONAL                    |
+
+#### Usage explanation
+
+`MANDATORY`: This field is required for proper operation of boudicca.
+`RECOMMENDED`: This field is not required, but it is one of the core features enabling event sorting and filtering for
+publishers.
+`OPTIONAL`: This field can be used if the information is available.
+`DEPRECATED`: This field should no longer be used.
+
+#### Registration enum values
+
+* `free`: a free event which neither requires registration nor a ticket
+* `registration`: an event which requires a free registration
+* `pre-sales-only`: an event which can only be entered when buying a ticket in advance
+* `ticket`: a paid event which requires a ticket
+
+#### "type" property
+
+type is an open field, so any value is permitted, but this is a list of common/known types:
+
+- concert
+- theatre
+- meetup
+- museum
+- museum_railway
+- model_railway
+- guided_tour
+- workshop
+- ...
+
+#### "tags" property
+
+tags is an open field, so any value is permitted, but this is a list of common/known tags:
+
+- ...
+- TODO
```

Comment:
todo

Reply:
yeah, that todo is open since the beginning of (boudicca) time, and i still don't know better :D